src/type-translator.ts

```diff
@@ -188,7 +188,12 @@ export class TypeTranslator {
     // NOTE: Though type.flags has the name "flags", it usually can only be one
     // of the enum options at a time.  This switch handles all the cases in
     // the ts.TypeFlags enum in the order they occur.
-    switch (type.flags) {
+    // NOTE: Some TypeFlags are marked "internal" in the d.ts but still show
+    // up in the value of type.flags.  This mask limits the flag checks to
+    // the ones in the public API.
```

Comment:
I extended the comment.  Linking on github doesn't work because the github d.ts is different than the version we depend on in this code.